### PR TITLE
Fix privilege separation directory for Fedora 34

### PR DESCRIPTION
**Pull request by 53c70r**

### CODE REVIEW

**Comment by laolux** [comment]
I just tried it and it fixed Fedora 34 for me. Thanks a lot!

File: 46sshd/module-setup.sh [hunk 1]
```diff
@@ -68,13 +68,14 @@ install() {
...
-    # /var/chroot/ssh  -> Void Linux
+    # /var/empty/sshd       -> Fedora, CentOS, RHEL
+    # /usr/share/empty.sshd -> Fedora >= 34
+    # /var/emtpy            -> Arch, OpenSSH upstream
+    # /var/lib/empty        -> Suse
+    # /run/sshd             -> Debian
+    # /var/chroot/ssh       -> Void Linux
     local d
-    for d in /var/empty/sshd /var/empty /var/lib/empty /run/sshd /var/chroot/ssh ; do
+    for d in /var/empty/sshd /usr/share/empty\.sshd /var/empty /var/lib/empty /run/sshd /var/chroot/ssh ; do
```
Comment (gsauthof) [inline 46sshd/module-setup.sh:78]: Hm, is this `\.` in `/usr/share/empty\.sshd` necessary?

Comment (53c70r) [reply 46sshd/module-setup.sh:78]: Yes.

Comment (gsauthof) [reply 46sshd/module-setup.sh:78]: So please update (via force-push) your pull request (i.e. remove the superfluous escape)  such that I can merge it.

**Comment by gsauthof** [comment]
@53c70r thanks for the pull request, @laolux  thank you for testing it.

**Comment by champtar** [comment]
Thanks for the fix !

I lost access to one of my kimsufi (ovh) server, they do not provide a KVM or serial but you can reboot on a rescue system.
Of course this rescue system is a bit too old and unable to open recent luks partition:
```
# cryptsetup luksOpen /dev/sdaX openX
Unsupported LUKS version 2.
```
As you just want to add a folder you don't need to rebuild the initramfs, you can just append the new folder:
```
# mkdir -p usr/share/empty.sshd
# echo usr/share/empty.sshd | cpio -H newc -o >> initramfs-5.11.16-300.fc34.x86_64.img
1 block
```
Hope this trick can help

@gsauthof if you can release ASAP this will avoid some pain to all Fedora 33 user updating to 34 ;)

**Comment by gsauthof** [comment]
> @gsauthof if you can release ASAP this will avoid some pain to all Fedora 33 user updating to 34 ;)

Just tagged the repository and triggered a Copr build for Fedora 34.

**Pull request merged**